.github/workflows/test.yml

```diff
@@ -17,20 +20,8 @@ jobs:
       uses: actions/setup-python@v4
       with:
         python-version: ${{ matrix.python-version }}
-
-    - name: Get pip cache dir
-      id: pip-cache
-      run: |
-        echo "::set-output name=dir::$(pip cache dir)"
-
-    - name: Cache
-      uses: actions/cache@v3
-      with:
-        path: ${{ steps.pip-cache.outputs.dir }}
-        key:
-          ${{ matrix.os }}-${{ matrix.python-version }}-v1-${{ hashFiles('**/setup.py') }}
-        restore-keys: |
-          ${{ matrix.os }}-${{ matrix.python-version }}-v1-
+        cache: pip
+        cache-dependency-path: setup.py
```

Comment:
Question: why do we need to specify `setup.py` as a cache-dependency-path?

Reply:
The usual idea is to specify which file(s) contains the dependencies to be cached.

Then, when that file changes, it invalidates the cache and creates a new one.

I don't think we actually have any dependencies, but we have to have something otherwise it defaults to `requirements.txt` and fails when it can't find it.

So `setup.py` is as good as any, and most likely of the existing files for any new dependencies to be added.

Reply:
I see. Probably, in the future, we can create `requirements-dev.txt` and put all dependencies there, but I think it's not necessary in this PR.